nex/friends-3ds/get_pid_by_lfc.go

```diff
@@ -26,4 +26,6 @@ func GetPrincipalIDByLocalFriendCode(err error, client *nex.Client, callID uint3
 	responsePacket.AddFlag(nex.FlagReliable)
 
 	globals.SecureServer.Send(responsePacket)
+
+	return 0
```

Comment:
Since this is stubbed right now we can just remove this entire response and return `nex.Errors.Core.NotImplemented`, keeping the comment